src/tests/unit/tests/injected/element-based-view-model-creator.test.ts

```diff
@@ -118,14 +134,9 @@ describe('ElementBasedViewModelCreator', () => {
             const unifiedResult = cloneDeep(exampleUnifiedResult);
             unifiedResult.status = testStatus;
             const ruleStub = { id: unifiedResult.ruleId } as UnifiedRule;
-            const unifiedRules = [ruleStub];
             const identifierStub = unifiedResult.identifiers['css-selector'];
             const decoratedResultStub = {} as DecoratedAxeNodeResult;
-            const scanResultStoreData = {
-                results: [unifiedResult],
-                rules: unifiedRules,
-                platformInfo: null,
-            } as UnifiedScanResultStoreData;
+            const ScanNodeResults = [{ ...unifiedResult, rule: ruleStub }];
```

Comment:
nit: casing on the variable
```suggestion
            const scanNodeResults = [{ ...unifiedResult, rule: ruleStub }];
```